### PR TITLE
Update the Atlassian SDK repositories based on latest info from

### PR DESCRIPTION
**Pull request by mihaics3**
https://developer.atlassian.com/server/framework/atlassian-sdk/install-the-atlassian-sdk-on-a-linux-or-mac-system/

Signed-off-by: Mihai Catalin Stefan <mihaics3@gmail.com>

### CODE REVIEW

**Comment by mihaics3** [comment]
This is the fix for https://github.com/pycontribs/jira/issues/690

**Comment by hdost** [comment]
@mihaics3 can you rebase your change off master?

**Comment by mihaics3** [comment]
@hdost I've rebased the change. Checks are in progress.

**Comment by hdost** [comment]
The PR looks muddled could you cherry pick your changes?

File: jira/resources.py [hunk 1]
```diff
@@ -590,7 +590,7 @@ def permalink(self):
 
     def __eq__(self, other):
         """Comparison method."""
-        return self.id == other.id
```
Comment (ssbarnea) [inline jira/resources.py]: This has nothing to do with this change.

**Comment by mihaics3** [comment]
I've cherry-picked the change and is pushed in [issue-690-1](https://github.com/mihaics3/jira/tree/issue-690-1) branch. Should I open a new pull request ?

**Comment by hdost** [comment]
Yes open another and we can close this one

**Comment by mihaics3** [comment]
@hdost please see https://github.com/pycontribs/jira/pull/714 for the new pull request. Thank you.

**Pull request closed without merging**